dt-posts/dt-posts.php

```diff
@@ -1807,8 +1807,8 @@ public static function get_post_tiles( $post_type, $return_cache = true ){
         }
         $tile_options = dt_get_option( 'dt_custom_tiles' );
         $default = [
-            'status' => [ 'label' => __( 'Status', 'disciple_tools' ), 'tile_priority' => 10 ],
-            'details' => [ 'label' => __( 'Details', 'disciple_tools' ), 'tile_priority' => 20 ]
+            'status' => [ 'default_label' => __( 'Status', 'disciple_tools' ), 'tile_priority' => 10 ],
```

Comment:
@micahmills Let's keep the default to be just `label` then the mobile will still know where to get the name of the tile.